apps/prairielearn/elements/pl-matching/pl-matching.css

```diff
@@ -47,3 +47,38 @@
   background-color: rgba(0, 0, 0, 0.03);
   margin-right: 0.5rem;
 }
+
+.pl-matching-container-right {
+  display: flex;
+}
+
+.pl-matching-container-right .pl-matching-statements {
+  flex: 0.2 0 25%;
```

Comment:
The `flex:` setting caused some weird results in narrow width scenarios, particularly with the container not using the whole width. I removed it (and in the similar settings below for `pl-matching-options`) and it looked better.

Reply:
Yes, it looks fine as it is. I've removed it